app/rdf/query-search.ts

```diff
@@ -183,9 +223,8 @@ export const searchCubes = async ({
       ||  (bound(?creatorLabel) && ${query
         .split(" ")
         .map((x) => icontains("?creatorLabel", x))
-        .join(" || ")})  
-
-          )`
+        .join(" || ")})
+      )`
```

Comment:
Cool to have only 1 query ! Do you remember the time savings ? Don't know why I separated it into to again 😓

Reply:
The removed DESCRIBE query was taking around 200ms, and the overall improvement is hard to measure yet, but e.g. currently on PROD the search query without any filters takes 1.5s, and in the preview deployment it's 0.15s 👀 But it also might depend on the deployment itself (locations of servers, etc), so I guess we'll fully see when it's deployed to TEST :)